addons/website_event_booth_sale/static/tests/tours/website_event_booth_sale_pricelists.js

```diff
@@ -0,0 +1,74 @@
+/** @odoo-module **/
+
+import tour from 'web_tour.tour';
+import { getPriceListChecksSteps } from 'website_event_booth_sale.tour.WebsiteEventBoothSaleTourMethods';
+
+tour.register('event_booth_sale_pricelists_different_currencies', {
+    test: true,
+    url: '/event',
+}, [
+    // Init: registering the booth
+    {
+        content: 'Open "Test Event Booths" event',
+        trigger: 'h5.card-title span:contains("Test Event Booths")',
+    },
+    {
+        content: 'Go to "Get A Booth" page',
+        trigger: 'li.nav-item a:has(span:contains("Get A Booth"))',
+    },
+    {
+        content: 'Select the booth',
+        trigger: '.o_wbooth_booths input[name="event_booth_ids"]',
+        run: function () {
+            $('.o_wbooth_booths input[name="event_booth_ids"]:lt(1)').click();
+            },
+    },
+    {
+        content: 'Confirm the booth by clicking the submit button',
+        trigger: 'button.o_wbooth_registration_submit',
+    },
+    {
+        content: 'Fill in your contact information',
+        trigger: 'input[name="contact_name"]',
+        run: function () {
+            $('input[name="contact_name"]').val('John Doe');
+            $('input[name="contact_email"]').val('jdoe@example.com');
+            },
+    },
+    {
+        content: 'Submit your informations',
+        trigger: 'button[type="submit"]',
+    },
+    /*
+    Test Case 1:
+    1) Change pricelist to "EUR With Discount Included"
+    2) Verify price of the booth -> 90.00
+    3) Verify price of the selected booth -> 90
+    4) Verify price in the cart -> 99.00
+    */
+    ...getPriceListChecksSteps("EUR With Discount Included", "Test Event Booths", "90.00", "90", "99.00"),
```

Comment:
Now that have another look at all the required values, I think this is a case where the code should be more explicit so that it removes the need for the comments (an error-prone second thing to maintain...)

So it could be something like 
```suggestion
    ...getPriceListChecksSteps(
        pricelistName="EUR With Discount Included",
        event="Test Event Booths", 
        expected_price_shown="90.00",
        expected_price_shown_selected="90.00",
        expected_total_price_in_cart="99.00"
    ),
```

Reply:
Ok I had to to "destructure" the function arguments to have what you wanted: kind of Python's keywords arguments
https://reactgo.com/destructuring-function-agruments-javascript/#destructuring-function-arguments-example